ghastly/simulation.py

```diff
@@ -0,0 +1,12 @@
+class Sim:
+    '''
+    Class for containing simulation-wide parameters and methods.
+    '''
+
+    def __init__(self, r_pebble, t_final, down_flow=True):
+        '''
+        Initializes the Sim class.
```

Comment:
Missing the doc strings for this function.

Reply:
I'll fix this alongside the other attribute that should have been removed